pkg/reconciler/route/config/store_test.go

```diff
@@ -0,0 +1,58 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package config
+
+import (
+	"context"
+	"fmt"
+	"testing"
+
+	"github.com/google/go-cmp/cmp"
+	"github.com/google/kf/pkg/kf/testutil"
+	logtesting "knative.dev/pkg/logging/testing"
+
+	. "knative.dev/pkg/configmap/testing"
+)
+
+func TestStoreLoadDefault(t *testing.T) {
+	store := NewDefaultConfigStore(logtesting.TestLogger(t))
+	routingConfig := FromContext(store.ToContext(context.Background())).Routing
+
+	t.Run("routing config defaults", func(t *testing.T) {
```

Comment:
nit: Given that there aren't any additional tests, could we not use the t.Run here?

Reply:
yep took that wrapper out